.gitignore

```diff
@@ -1,4 +1,5 @@
-/attachments/
+/attachments/*
+!/attachments/.gitkeep
```

Comment:
I was using [this StackOverflow](http://stackoverflow.com/a/19005115) article as an example, and it discouraged that.